### PR TITLE
Feature/CLDN-1427

### PR DESCRIPTION
**Pull request by reesercollins**
### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
image

### ADDITIONAL INFORMATION

- [x] Has associated issue: https://cccs.atlassian.net/browse/CLDN-1427
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/CccsGrid.tsx [hunk 1]
```diff
@@ -234,6 +235,13 @@ export default function CccsGrid({
     params.columnApi.autoSizeColumns(allColumnIds.slice(0, 100), false);
   }
 
+  function setSearch(e: ChangeEvent<HTMLInputElement>) {
+    const target = e.target as HTMLInputElement;
+    e.preventDefault();
+    console.log(target.value);
```
Comment (cccs-joel) [inline superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/CccsGrid.tsx:241]: Do we still need console.log ?

Comment (reesercollins) [reply superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/CccsGrid.tsx:241]: Oops!

File: superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/CccsGrid.tsx [hunk 2]
```diff
@@ -244,6 +252,22 @@ export default function CccsGrid({
 
   return (
     <div style={{ width, height }} className="ag-theme-balham">
+      <div className="form-inline" style={{ paddingBottom: "0.5em" }}> 
+        <div className="row">
+          <div className="col-sm-6"></div>
```
Comment (cccs-joel) [inline superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/CccsGrid.tsx:256]: I imagine this is just a spacer.

Comment (reesercollins) [reply superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/CccsGrid.tsx:256]: Partially, yes. But the form-inline and grid also provide styling for the input element.

File: superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/CccsGrid.tsx [hunk 3]
```diff
@@ -258,6 +282,8 @@ export default function CccsGrid({
         onRangeSelectionChanged={onRangeSelectionChanged}
         onSelectionChanged={onSelectionChanged}
         rowData={rowData}
+        cacheQuickFilter={true}
```
Comment (cccs-joel) [inline superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/CccsGrid.tsx:284]: Glad to see Ag grid had that feature already built in!

**Review by cccs-rc — APPROVED** [review]

**Pull request merged**